model/openapi.yml

```diff
@@ -542,129 +607,146 @@ components:
           type: string
         action_id:
           description: The action ID.
           type: string
-        action_input_type:
-          description: The input_type of the action for input actions.
-          type: string
-        policy_id:
-          deprecated: true
-          description: Not used by the fleet-server.
-          type: string
-        stream_id:
-          deprecated: true
-          description: Not used by the fleet-server.
-          type: string
-        timestamp:
-          description: The timestamp of the acknowledgement event. Has the format of "2006-01-02T15:04:05.99999-07:00"
-          type: string
-          #format: date-time
         message:
           description: An acknowlegement message. The elastic-agent inserts the action ID and action type into this message.
           type: string
-        payload:
-          deprecated: true
-          description: |
-            An embedded JSON object that contains additional information for the fleet-server to process.
-            Defined as a json.RawMessage in both the fleet-server and the elastic-agent.
-
-            Is currently used by UPGRADE actions to signal retries.
-            If the error attribute is non empty payload is checked for `retry: bool` and `retry_attempt: int`.
-            If retry is true, fleet-serve will mark the agent as retrying, if it's false the upgrade will be marked as failed.
-
-            Additional action status information can be provided in the data attribute.
-          type: string
-          format: application/json
-          x-go-type: json.RawMessage
-        started_at:
-          description: The time at which the action was started. Used only when acknowledging input actions.
-          type: string
-          #format: date-time
-        completed_at:
-          description: The time at which the action was completed. Used only when acknowledging input actions
-          type: string
-          #format: date-time
-        action_data:
-          description: The action data for the input action being acknowledged.
-          type: string
-          format: application/json
-          x-go-type: json.RawMessage
-        action_response:
-          description: The action response for the input action being acknowledged.
-          type: string
-          format: application/json
-          x-go-type: json.RawMessage
-        data:
-          description: |
-            An embedded JSON object that has the data about the ack.
-
-            Used by REQUEST_DIAGNOSTICS actions.
-            Contains a `upload_id` attribute used to communicate the successfullly uploaded diagnostics ID.
+        timestamp:
+          description: The timestamp of the acknowledgement event. Has the format of "2006-01-02T15:04:05.99999-07:00"
           type: string
-          format: application/json
-          x-go-type: json.RawMessage
+          format: date-time
```

Comment:
Note that we are now forcing serialization for the `timestamp`, `started_at`, and `completed_at` properties. additionally the `timestamp` attribute will now be set in the results as well